lib/wicked_pdf.rb

```diff
@@ -321,7 +325,7 @@ def parse_others(options)
                                   :disable_smart_shrinking,
                                   :use_xserver,
                                   :no_background], '', :boolean)
-      r += make_options(options, [:no_stop_slow_scripts], '', nil)
+      r += make_options(options, [:no_stop_slow_scripts], '', :boolean)
```

Comment:
Could you please put `:no_stop_slow_scripts` in the boolean array in the line above this one, then delete this line?

It should have the same result.

Thank you for your contribution!